test/testrunner.py

```diff
@@ -198,6 +199,13 @@ def main():
         run_sagemaker_local_tests(
             [image for image in standard_images_list if not (("tensorflow-inference" in image and "py2" in image) or ("eia" in image))]
         )
+        for image in standard_images_list:
+            if "eia" in image:
```

Comment:
I think the issue is we could not finish a task without report file. It might also fall into this situation that only non-ei tf-inf-py2 images are built and tested. Could you also apply your work to non-ei-tf-inf-py2 images? Also utilize it to line 196

Reply:
Reorganize them as a function:
```
testing_image_list = [image for image in standard_images_list if not (("tensorflow-inference" in image and "py2" in image) or ("eia" in image))]
run_sagemaker_local_tests(testing_image_list) # or run_sagemaker_remote_tests(testing_image_list)
for image in standard_images_list:
    if image not in testing_image_list:
      generate_empty_report()
```
```

Reply:
There is two condition we are checking -
1) If it is eia images, which is taken care by this change.
2) If it is py2 tf inference image, in that case , we are build both py3 and py2. Hence it passes.

Reply:
> Reorganize them as a function:
> 
> ```
> testing_image_list = [image for image in standard_images_list if not (("tensorflow-inference" in image and "py2" in image) or ("eia" in image))]
> run_sagemaker_local_tests(testing_image_list) # or run_sagemaker_remote_tests(testing_image_list)
> for image in standard_images_list:
>     if image not in testing_image_list:
>       generate_empty_report()
> ```

I can try this out.

Reply:
+1 on function